README.md

```diff
@@ -96,6 +96,23 @@ You can also install the dev requirements by adding `--with dev` or the extras
 for say mujoco and acceleration by [envpool](https://github.com/sail-sg/envpool)
 by adding `--extras mujoco envpool`
 
+Available extras are:
+- `atari` (for Atari environments)
+- `box2d` (for Box2D environments)
+- `classic_control` (for classic control (discrete) environments)
+- `mujoco` (for MuJoCo environments)
+- `mujoco-py` (for legacy mujoco-py environments[^1])
+- `pybullet` (for pybullet environments)
+- `robotics` (for gymnasium-robotics environments)
+- `vizdoom` (for ViZDoom environments)
+- `envpool` (for envpool integration)
+- `dev` (for development requirements)
```

Comment:
Isn't that a group rather than an extra?

Reply:
If you're referring to 'dev', then you're right. 'robotics' and 'vizdoom' should be extras (in the same fashion as 'atari' and 'mujoco' before). Should I just remove it from readme?